### DISCUSSION TITLE
Problem with inicialization

### DISCUSSION THREAD

**Discussion by jqnag8** [zed-industries/zed#14702] [Feature Requests]
Hi guys! When I want to initialize Zed, the application starts, shows a prompt in the taskbar but it doesn't show any GUI to start working. Any idea how to fix it? I tried reinstalling Vulkan and my gpu drivers but it keeps giving that error.

I am running it on:
- GTX 1050 ti
- Xubuntu 24.04

Thank you!!
![Screenshot_2024-07-18_00-07-11](https://github.com/user-attachments/assets/1185edbe-cc80-4933-83f0-158a4a505503)

**Comment by SomeoneToIgnore** [comment]
I'd check https://github.com/zed-industries/zed/issues/14204 in case you might have some outdated GPU drivers or related issues.
Alternatively, I'd try to nuke the DB in the local directory: https://github.com/zed-industries/zed/issues/14430

**Reply by jqnag8** [reply]
Thanks, I'll do that

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
